MetaMorpheus/Test/ProteinGroupTest.cs

```diff
@@ -217,13 +217,13 @@ public static void TestModificationInfoListInProteinGroupsOutput()
             int totalNumberOfMods = proteins.Sum(p => p.OneBasedPossibleLocalizedModifications.Count + p.SequenceVariations.Sum(sv => sv.OneBasedModifications.Count));
 
             //tests that modifications are being done correctly
-            Assert.AreEqual(0, totalNumberOfMods);
```

Comment:
The change of an expected value with the comment above, "test that mods are being done correctly" is worrisome

Reply:
idk it is 8 now. maybe this is old news.